waspc/data/Generator/templates/server/src/core/auth/prismaMiddleware.js

```diff
@@ -60,12 +60,13 @@ export const registerAuthMiddleware = (prismaClient) => {
   registerPasswordHashing(prismaClient)
 }
 
-const userValidations = [
-  { validates: PASSWORD_FIELD, message: 'password must be present', validator: password => !!password },
-  { validates: PASSWORD_FIELD, message: 'password must be at least 8 characters', validator: password => password.length >= 8 },
-  { validates: PASSWORD_FIELD, message: 'password must contain a number', validator: password => /\d/.test(password) },
-]
+const userValidations = []
 
+{=# isEmailOrUsernameAndPasswordAuthEnabled =}
+userValidations.push({ validates: PASSWORD_FIELD, message: 'password must be present', validator: password => !!password })
```

Comment:
@shayneczyzewski We need password validation for both username and email & password auth method.

Maybe pass into the template `isPasswordFieldValidated` or similar to encapsulate that logic.

Reply:
My bad, you are checking both!

I didn't read carefully the variable name `isEmailOrUsernameAndPasswordAuthEnabled` but I would still maybe rename it :)